index.html

```diff
@@ -1420,16 +1422,20 @@ <h3>Multi-Cloud</h3>
     </section>
 
     <section id="virtual-thing">
-      <h3>Virtual Things</h3>    
-      <section class="ednote" title="TODO">
-      <h3>TODO</h3>
-      Create a description and a diagram for a <a>virtual thing</a> scenario,
-      as described in the use case for 
-      <a href="https://w3c.github.io/wot-usecases/#USE-CASES/Retail-virtual-thing">Virtual Thing</a>
+      <h3>Virtual Things</h3>   
+	<p>
+	 A virtual thing is an abstraction of a physical entity that does not have a direct physical counterpart.
+	 It can be used to describe abstract interfaces with a TD that are decoupled from individual devices. 
+	 The purpose is to decouple a functional interface that defines the behavior
+	 from the specific implementations of devices among several manufacturers.
+	 The interface remains stable and is independent from the actual implementation of things behind that interface.
+	</p> 
+	<p>
+	  The virtual thing guarantees that 
```

Comment:
Who gives this guarantee? It sounds a little too "serious" for a document/chapter that is this "loose".

Reply:
Please elaborate what you mean with "too serious"

Reply:
I think the issue here is that the spec is making a guarantee but is not in a position to enforce it.  We could say that *one* use case is to provide a stable interface as a wrapper to another service that may be subject to change.  It's an application of virtual things, not its sole purpose or something we can guarantee.  I could also have a virtual thing that is a software service that is changing all the time.  It may not be a good idea, but I might want to do it for various reasons.

Reply:
Well this section is non normative so we should not talk about any guarantees